platform/amba-alz/architecture_definitions/amba.alz_architecture_definition.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/Azure-Landing-Zones-Library/main/schemas/architecture_definition.json",
+  "name": "alz",
+  "management_groups": [
+    {
+      "archetypes": [
+        "root"
```

Comment:
change the amba archetype name to ambaroot otherwise you'll try to overwrite alz's root archetype and this will fail...

Reply:
Then add both archetypes to this list...

```text
[
  "root",
  "ambaroot"
]
```